app/src/main/java/com/example/candyshop/Navigation.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.candyshop
+
+import androidx.compose.animation.AnimatedContentTransitionScope
+import androidx.compose.animation.core.tween
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+
+@Composable
+fun Navigation() {
+    val navController = rememberNavController()
+    val time = 500
```

Comment:
maybe we could change this one and put on top of the function: 

private const val TWEEN_TIME = 500

in this way this const value is better optimised, naming is more clear why we are creating it and for const values we are using the same notation as in our app